block/stream/output/state_changes.proto

```diff
@@ -353,6 +353,16 @@ enum StateIdentifier {
      */
     STATE_ID_ROSTERS = 28;
 
+    /**
+    * A state identifier for scheduled transaction expiration.
+    */
+    STATE_ID_SCHEDULE_IDS_BY_EXPIRY = 29;
+
+    /**
+     * A state identifier for scheduled transaction deduplication.
+     */
+    STATE_ID_SCHEDULE_ID_BY_EQUALITY = 30;
+
```

Comment:
These should **replace** enumerations 15 and 16.  Block streams aren't published yet, so we can still make that change.

Reply:
Those values are used in the code for migration to work. Should not replace them.

Reply:
That's odd, this enumeration is specific to the block stream and _should not_ be used in state.

Reply:
There are many code already use those values. The code won't even compile if we remove those values. This PR is to just sync the changes from hedera-services hapi directory to hedera-protobufs. We had our internal services meeting is to make code changes in the hedera-services and sync the changes to hedera-protobufs.

Reply:
I get that, it's more that we _should_ be able to still rewrite/update these enumerations until block streams is live.
Using block stream definitions in state is going to create all manner of problems eventually, because blocks streams need to change _independent_ of the consensus node.

Reply:
That's a big topic to discuss with Schorodinger Team who works on Block Steams. This PR is just to sync my changes from hedera-services PR 16144.

Reply:
Sure, the proto can be changed in a later PR to clean up unnecessary items and adjust the numbering accordingly.
It would be helpful if you would initiate any necessary discussions when time permits.

Reply:
I will mention this at our (Blockchain Wizards team) next daily this Thur. (Skipped tomorrow because of another meeting). Services has two teams on different daily.